Source/WTF/wtf/threads/Signals.cpp

```diff
@@ -203,6 +206,157 @@ kern_return_t catch_mach_exception_raise_state_identity(mach_port_t, mach_port_t
     return KERN_FAILURE;
 }
 
+#ifdef EXCEPTION_IDENTITY_PROTECTED
+
+static thread_act_t threadIDGetThreadPort(uint64_t threadID)
+{
+    kern_return_t kr;
+    mach_msg_type_number_t i, thcnt;
```

Comment:
This isn't standard WebKit style. Typically we'd have declare `i` in the for loop below and we would spell out `threadCount`.